app/src/main/java/com/kickstarter/viewmodels/projectpage/ProjectOverviewViewModel.kt

```diff
@@ -608,6 +613,12 @@ interface ProjectOverviewViewModel {
 
             setSuccessfulProjectStateView = project
                 .filter { it.isSuccessful }
+                .filter { !(it.postCampaignPledgingEnabled() ?: false) && !(it.isInPostCampaignPledgingPhase() ?: false) }
```

Comment:
Can we add tests on `ProjectOverviewViewModelTest` to make sure this newly added logic is covered?